Jenkinsfile

```diff
@@ -83,6 +83,8 @@ pipeline {
 			}
                     }
                     steps {
+			echo 'Build all the examples in the book'
+			sh 'mdbook test book -L target/debug/deps'
```

Comment:
hm, i suspect this might have to go after `cargo test --all`, but i'll check with bors